lighthouse-core/report/html/report-styles.css

```diff
@@ -143,6 +149,9 @@
     --circle-size-big: 96px;
     --circle-size: 72px;
     --header-padding: 16px 0 16px 0;
+    --plugin-badge-size: 28px;
```

Comment:
also once plugin-badge-size is sized off circle-size we dont need this secondary definition. ;)

Reply:
done